app/locale/da.coffee

```diff
@@ -279,55 +279,55 @@ module.exports = nativeDescription: "dansk", englishDescription: "Danish", trans
 #    teacher_button: "For Teachers"
 #    get_more_codecombat: "Get More CodeCombat"
 
-#  code:
-#    if: "if" # Keywords--these translations show up on hover, so please translate them all, even if it's kind of long. (In the code editor, they will still be in English.)
-#    else: "else"
-#    elif: "else if"
-#    while: "while"
-#    loop: "loop"
-#    for: "for"
-#    break: "break"
-#    continue: "continue"
-#    pass: "pass"
-#    return: "return"
-#    then: "then"
-#    do: "do"
-#    end: "end"
-#    function: "function"
-#    def: "define"
-#    var: "variable"
-#    self: "self"
-#    hero: "hero"
-#    this: "this"
-#    or: "or"
-#    "||": "or"
-#    and: "and"
-#    "&&": "and"
-#    not: "not"
-#    "!": "not"
-#    "=": "assign"
-#    "==": "equals"
-#    "===": "strictly equals"
-#    "!=": "does not equal"
-#    "!==": "does not strictly equal"
-#    ">": "is greater than"
-#    ">=": "is greater than or equal"
-#    "<": "is less than"
-#    "<=": "is less than or equal"
-#    "*": "multiplied by"
-#    "/": "divided by"
-#    "+": "plus"
-#    "-": "minus"
-#    "+=": "add and assign"
-#    "-=": "subtract and assign"
-#    True: "True"
-#    true: "true"
-#    False: "False"
-#    false: "false"
-#    undefined: "undefined"
-#    null: "null"
-#    nil: "nil"
-#    None: "None"
+  code:
+    if: "hvis" # Keywords--these translations show up on hover, so please translate them all, even if it's kind of long. (In the code editor, they will still be in English.)
+    else: "ellers"
+    elif: "ellers hvis"
+    while: "mens"
+    loop: "løkke"
+    for: "for"
+    break: "afbryd"
+    continue: "fortsæt"
+    pass: "spring over"
+    return: "returnér"
+    then: "så"
+    do: "gør"
+    end: "slut"
+    function: "funktion"
+    def: "definér"
+    var: "variabel"
+    self: "selv"
+    hero: "helt (din kriger)"
+    this: "denne"
+    or: "eller"
+    "||": "eller"
+    and: "og"
+    "&&": "og"
+    not: "ikke"
+    "!": "ikke"
+    "=": "tildel"
+    "==": "lig med"
+    "===": "præcis lig med"
+    "!=": "ikke lig med"
+    "!==": "ikke præcis lig med"
+    ">": "er større end"
+    ">=": "er større end eller lig med"
+    "<": "er mindre end"
+    "<=": "er mindre end eller lig med"
+    "*": "ganget med"
+    "/": "divideret med"
+    "+": "plus"
+    "-": "minus"
+    "+=": "læg sammen og tildel"
+    "-=": "træk fra og tildel"
+    True: "Sandt"
+    true: "sandt"
+    False: "Falsk"
+    false: "falsk"
+    undefined: "udefineret"
+    null: "null (udefineret)"
```

Comment:
Would it make sense to do the same in English, to explain that `null` and `nil` means undefined?
Or are `null` and `nil` common enough words in English?